### PR TITLE
Update health values for battery

### PR DESCRIPTION
**Pull request by prashanth-intel**
add sysfs read for charge_full and charge_full_design
to update battery health

Tracked-On: OAM-102730
Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>

### CODE REVIEW

**Review by shyjumon-n — APPROVED** [review]
LGTM

File: health/2.1/HealthImpl.cpp [hunk 1]
```diff
@@ -58,9 +59,11 @@ void HealthImpl::UpdateHealthInfo(HealthInfo* health_info) {
     convertFromHealthInfo(health_info->legacy.legacy, &props);
     healthd_board_battery_update(&props);
     convertToHealthInfo(&props, health_info->legacy.legacy);
-    health_info->batteryCapacityLevel = android::hardware::health::V2_1::BatteryCapacityLevel::UNKNOWN;
-    health_info->batteryChargeTimeToFullNowSeconds = 0;
-    health_info->batteryFullChargeDesignCapacityUah = 0;
+    if ( get_battery_mediation_present() == false){
```
Comment (vilasrk) [inline health/2.1/HealthImpl.cpp:62]: Add space after round bracket.

File: health/2.1/libhealthd/healthd_board_default.cpp [hunk 2]
```diff
@@ -44,6 +45,18 @@ static void parse_battery_status(uint8_t *status)
         s_props.batteryStatus = android::BATTERY_STATUS_FULL;
 }
 
+static void update_battery_health(struct monitor_pkt *mpkt)
+{
+        if( (mpkt->charge_full == 0) || (mpkt->charge_full_design == 0) ) {
```
Comment (vilasrk) [inline health/2.1/libhealthd/healthd_board_default.cpp:50]: Indent properly.

File: health/2.1/libhealthd/healthd_board_default.cpp [hunk 3]
```diff
@@ -44,6 +45,18 @@ static void parse_battery_status(uint8_t *status)
...
+            s_props.batteryHealth = android::BATTERY_HEALTH_DEAD;
+        }
+        else if ( (((mpkt->charge_full)*100)/mpkt->charge_full_design) >= 80 ) {
```
Comment (vilasrk) [inline health/2.1/libhealthd/healthd_board_default.cpp:53]: Indent properly.

**Review by vilasrk — APPROVED** [review]

**Review by balajim001 — APPROVED** [review]
LGTM

**Pull request merged**